scripts/write_deployinst_input.py

```diff
@@ -217,63 +217,39 @@ def reactor_render(list, output_file, is_cyborg=False):
     pwr_template = read_template(template_path.replace('[reactor]', 'pwr'))
     mox_reactor_template = read_template(template_path.replace('[reactor]', 'mox'))
     candu_template = read_template(template_path.replace('[reactor]', 'candu'))
+
+    spec_dict = {}
+    # spec_dict[ 'reactor_type'] = [template, assem_size, n_assme_core(for model), n_assem_batch(for model)]
```

Comment:
I'd recommend a more informative dictionary structure (or even a class, given this level of detail per reactor!)

```
ap1000_spec = {'template' : 'pwr_template',
               'kg_per_assembly': 467.0,
               'num_assemblies_per_core': 157.0,
               'num_assemblies_per_batch': 52}
phwr_spec = {'template' : 'candu_template',
             'kg_per_assembly': int(8000.0 / 473.0),
             'num_assemblies_per_core': 473.0 / 50.0,
             'num_assemblies_per_batch': 60.0}

reactor_specs = {'AP1000' : ap1000_spec,
                 'PHWR' : phwr_spec}

```